src/AnalysisPrograms/Recognizers/Birds/CisticolaExilis.cs

```diff
@@ -0,0 +1,152 @@
+
+            // ################### POST-PROCESSING of EVENTS ###################
+            // Following two commented lines are different ways of casting lists.
+            //var newEvents = spectralEvents.Cast<EventCommon>().ToList();
+            //var spectralEvents = events.Select(x => (SpectralEvent)x).ToList();
+
+            if (combinedResults.NewEvents.Count == 0)
+            {
+                CisticolaLog.Debug($"Return zero events.");
+                return combinedResults;
+            }
+
+            // 1: Filter the events for duration in seconds
+            var minimumEventDuration = 0.1;
+            var maximumEventDuration = 0.25;
+            combinedResults.NewEvents = EventExtentions.FilterOnDuration(combinedResults.NewEvents, minimumEventDuration, maximumEventDuration);
+            CisticolaLog.Debug($"Event count after filtering on duration = {combinedResults.NewEvents.Count}");
+
+            // 2: Filter the events for bandwidth in Hertz
+            double average = 600;
+            double sd = 150;
+            double sigmaThreshold = 3.0;
+            combinedResults.NewEvents = EventExtentions.FilterOnBandwidth(combinedResults.NewEvents, average, sd, sigmaThreshold);
+            CisticolaLog.Debug($"Event count after filtering on bandwidth = {combinedResults.NewEvents.Count}");
+
+            // 3: Filter on COMPONENT COUNT in Composite events.
+            //int maxComponentCount = 2;
+            //combinedResults.NewEvents = EventExtentions.FilterEventsOnCompositeContent(combinedResults.NewEvents, maxComponentCount);
+            //CisticolaLog.Debug($"Event count after filtering on component count = {combinedResults.NewEvents.Count}");
+
+            //combinedResults.NewEvents = FilterEventsOnFrequencyProfile(combinedResults.NewEvents);
+
+            //UNCOMMENT following line if you want special debug spectrogram, i.e. with special plots.
+            //  NOTE: Standard spectrograms are produced by setting SaveSonogramImages: "True" or "WhenEventsDetected" in UserName.SpeciesName.yml config file.
+            //GenericRecognizer.SaveDebugSpectrogram(territorialResults, genericConfig, outputDirectory, audioRecording.BaseName);
+            return combinedResults;
+        }
+
+        /*
+        /// <summary>
+        /// Summarize your results. This method is invoked exactly once per original file.
+        /// </summary>
+        public override void SummariseResults(
+            AnalysisSettings settings,
+            FileSegment inputFileSegment,
+            EventBase[] events,
+            SummaryIndexBase[] indices,
+            SpectralIndexBase[] spectralIndices,
+            AnalysisResult2[] results)
+        {
+            // No operation - do nothing. Feel free to add your own logic.
+            base.SummariseResults(settings, inputFileSegment, events, indices, spectralIndices, results);
+        }
+        */
+
+        /// <inheritdoc cref="CisticolaExilisConfig"/> />
```

Comment:
what documentation is this inheriting from? It currently says it is inheriting from itself?
```suggestion
```